### PR TITLE
updated authroized keys to remove the append

### PR DESCRIPTION
**Pull request by JasonIamAUnixAdmin**
The appending of the authorized_keys was causing issues. Also cleaned out old comment from a prior update

### CODE REVIEW

File: dr/base/init.sls [hunk 1]
```diff
@@ -84,14 +79,13 @@ install-{{pippkg}}:
...
-    - mode: 755
-    - makedirs: true
-    - sources:
-      - salt://dr/home/authorized_keys
+{% for sshkey in pillar.dr.authorized_keys %}
+users_authorized_keys.{{ loop.index }}:
+  ssh_auth:
+    - present
+    - user: {{ pillar.__stackdio__.username }}
+    - name: {{ sshkey }}
```
Comment (cpenner461) [inline dr/base/init.sls]: Any benefit to using `names` here and just giving it the `pillar.dr.authorized_keys`?

**Comment by JasonIamAUnixAdmin** [comment]
- names:   needs to be fed one key per line, so I don't think it is a good fit here.

**Comment by JasonIamAUnixAdmin** [comment]
I was wrong. Cleaned up syntax a bunch.

**Comment by cpenner461** [comment]
:+1:

**Pull request merged**